src/main/java/com/epam/reportportal/service/ReportPortalErrorHandler.java

```diff
@@ -82,10 +84,8 @@ private void handleError(Response<ByteSource> rs) throws RestEndpointIOException
 				//ok, it's known ReportPortal error
 				throw new ReportPortalException(statusCode, statusMessage, errorRS);
 			} else {
-
 				if (isNotJson(rs)) {
-
-					throw new InternalReportPortalClientException("Report portal is not functioning correctly. Response is not json: " + rs.getReason());
+                    throw new InternalReportPortalClientException(format("Report portal is not functioning correctly. Response is not json. Uri: [%s]; statusCode: [%s]; statusMessage: [%s];", rs.getUri(), statusCode, statusMessage));
```

Comment:
Please keep our formatting rules, you can find our formatter here: https://github.com/reportportal/reportportal/blob/master/idea_formatting_profile.xml

And please implement a unit-test on that. We are not pushing anything into develop without unit-tests since January 2020.